lambda/translcore.ml

```diff
@@ -1020,7 +1222,7 @@ and transl_apply ~scopes
       (fun (_, arg) ->
          match arg with
          | Omitted _ as arg -> arg
-         | Arg exp -> Arg (transl_exp ~scopes exp))
+         | Arg exp -> Arg (transl_exp ~scopes None exp))
```

Comment:
I've added a comment to that effect here (and in the other places you suggested), but I wouldn't want to give a false impression that every place this assumption is made is commented.  It will be work to think through what needs to change when we add non-value args/returns.